programs-remote-tests/tests/acceptance/_bootstrap.php

```diff
@@ -4,7 +4,7 @@
 if (getenv('CIRCLE_BUILD_NUM')) {
     $url = 'https://tests.secure.retreat.guru';
 } else {
-    $url = 'http://tests.programs.dev';
+    $url = 'http://tests.qa0.rbgapp.com';
```

Comment:
Same thing as above.  Not too sure how comfortable I am with exposing the locations of our QA environments, but I might just be paranoid.

Reply:
I did this with eli because we couldn't figure out how to get the rgconnect to connect to the test.programs.test.
Docker issues.

This way we can point the rgconnect to a branch other than master

Reply:
see env comment above